File: Day05/Hard/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    bool doesAliceWin(string s) {
        for (char c : s) {
            if (c=='a' || c=='e' || c=='i' || c=='o' || c=='u') {
                return true;
            }
        }
        return false;
    }
};

int main() {
    string s;
    cin >> s;
    Solution sol;
    cout << (sol.doesAliceWin(s) ? "true" : "false") << endl;
    return 0;
}

File: Day05/Hard/Solution.java
import java.util.*;

class Solution {
    public boolean doesAliceWin(String s) {
        for (char c : s.toCharArray()) {
            if (c=='a' || c=='e' || c=='i' || c=='o' || c=='u') {
                return true;
            }
        }
        return false;
    }
}

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        Solution sol = new Solution();
        System.out.println(sol.doesAliceWin(s));
        sc.close();
    }
}
